distribution/docker/Dockerfile

```diff
@@ -17,6 +17,7 @@
 # under the License.
 #
 
+ARG JDK_VERSION=11
```

Comment:
we should keep the arguments in line with the file upstream, and instead pass the version through the docker build arguments.

Reply:
Updated it to be in sync with the upstream

Reply:
@harinirajendran did we ever update the docker build to pass in the JDK version as a build argument?

Reply:
Looks like we added it in this PR
https://github.com/confluentinc/druid/pull/76/files

Reply:
yep, thanks I eventually found it